zerver/lib/zcommand.py

```diff
@@ -9,12 +9,12 @@
 
 def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]:
     def change_mode_setting(
-        command: str, switch_command: str, setting: str, setting_value: int
+        setting_name: str, switch_command: str, setting: str, setting_value: int
```

Comment:
Any suggestion for variable name here? `setting_name` is confusing but was not able to think of any other at that point.

Reply:
Yeah I can't come up with something better; it's a pretty contained description in this file, so I'm OK with letting it be.